holoviews/core/data/__init__.py

```diff
@@ -15,13 +15,23 @@
 from .grid import GridInterface
 from .ndelement import NdElementInterface
 
+datatypes = ['array', 'dictionary', 'grid', 'ndelement']
+
```

Comment:
So I can populate it depending on the availability of the various data backends. Happy to change the way this is handled though.